### PR TITLE
Plans: Fix plans view alignment issues with personal plan enabled

### PR DESCRIPTION
**Pull request by apeatling**
Before:

4isot3w6eu

After:

screen shot 2016-07-06 at 11 18 47 am

Confirmed working correctly in NUX and on plans page with .com sites and Jetpack sites.

Test live: https://calypso.live/?branch=fix/4-plan-view-alignment

### CODE REVIEW

File: client/signup/steps/plans/index.jsx [hunk 1]
```diff
@@ -185,11 +187,13 @@ module.exports = React.createClass( {
...
 	render: function() {
-		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' ),
+			personalPlanTestEnabled = abtest( 'personalPlan' ) === 'show' && isEnabled( 'plans/personal-plan' );
 
 		const classes = classNames( 'plans plans-step', {
 			'has-no-sidebar': true,
-			'is-wide-layout': showPlanFeatures
+			'is-wide-layout': showPlanFeatures,
+			'personal-plan-enabled': personalPlanTestEnabled
```
Comment (gwwar) [inline client/signup/steps/plans/index.jsx]: Minor, but our modifier classes usually begin with an `is` or `has`

**Comment by gwwar** [comment]
screen shot 2016-07-06 at 11 40 02 am
I think we can ignore this for now, but the popular banner does wrap to two lines in smaller viewports.

**Comment by gwwar** [comment]
👍 Left a few notes on naming, but looks good to 🚢

**Comment by apeatling** [comment]
Updated class names based on the above feedback. :+1:

**Pull request merged**